processing/src/main/java/io/druid/segment/serde/ColumnPartSerde.java

```diff
@@ -39,7 +39,19 @@
 })
 public interface ColumnPartSerde
 {
-  public long numBytes();
-  public void write(WritableByteChannel channel) throws IOException;
-  public ColumnPartSerde read(ByteBuffer buffer, ColumnBuilder builder, ColumnConfig columnConfig);
+  public interface Serializer
```

Comment:
in general, druid code base keeps nested class/interface definitions in the end. it will be good to do same in here and other places.